implementation/src/main/java/io/smallrye/jwt/config/JWTAuthContextInfoProvider.java

```diff
@@ -252,6 +266,7 @@ private static JWTAuthContextInfoProvider create(String publicKey, String public
         contextInfo.setExpGracePeriodSecs(expGracePeriodSecs.orElse(null));
         contextInfo.setJwksRefreshInterval(jwksRefreshInterval.orElse(null));
         SmallryeJwtUtils.setWhitelistAlgorithms(contextInfo, whitelistAlgorithms);
+        contextInfo.setExpectedAudience(expectedAudience.orElse(null));
```

Comment:
@MikeEdgar can you please add a get method alongside other getters, for the Thorntail extending this provider to check it if needed

Reply:
Added